i18n/po.go

```diff
@@ -0,0 +1,186 @@
+package i18n
```

Comment:
Very go of you to write this from scratch. :)

Reply:
I actually did shop around but didn't love any of the options.. and figured the subset we need isn't that complex 🤷‍♂

Reply:
To be clear I did the same and agree with your decision. :)

Reply:
At least one of the ones I looked at didn't even support hacking together your own PO file